src/seedu/addressbook/ui/TextUi.java

```diff
@@ -162,7 +135,8 @@ private static String getIndexedListForViewing(List<String> listItems) {
      * @param visibleIndex visible index for this listing
      */
     private static String getIndexedListItem(int visibleIndex, String listItem) {
-        return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
+    	String message = Formatter.getIndexedListItemFormat(visibleIndex, listItem);
+        return message;
```

Comment:
Good job trying to extract as much as you can to the Formatter class. Note that it may not be clear if something belongs in the Formatter class or the TextUi class. It depends on how you define the responsibility of the Formatter class.